docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -232,25 +237,22 @@ and then you will run the [upgrade cluster command]({{< relref "baremetal-upgrad
   ✅ validate immutable fields
   🎉 all cluster upgrade preflight validations passed
   Performing provider setup and validations
-  Pausing EKS-A cluster controller reconcile
-  Pausing Flux kustomization
-  GitOps field not specified, pause flux kustomization skipped
-  Creating bootstrap cluster
-  Installing cluster-api providers on bootstrap cluster
-  Moving cluster management from workload to bootstrap cluster
-  Upgrading workload cluster
-  Moving cluster management from bootstrap to workload cluster
-  Applying new EKS-A cluster resource; resuming reconcile
-  Resuming EKS-A controller reconciliation
+  Ensuring etcd CAPI providers exist on management cluster before upgrade
+  Pausing GitOps cluster resources reconcile
+  Upgrading core components
+  Backing up management cluster's resources before upgrading
+  Upgrading management cluster
   Updating Git Repo with new EKS-A cluster spec
-  GitOps field not specified, update git repo skipped
   Forcing reconcile Git repo with latest commit
-  GitOps not configured, force reconcile flux git repo skipped
-  Resuming Flux kustomization
-  GitOps field not specified, resume flux kustomization skipped
+  Resuming GitOps cluster resources kustomization
+  Writing cluster config file
+  🎉 Cluster upgraded!
+  Cleaning up backup resources
   ```
   
-  During the upgrade process, EKS Anywhere pauses the cluster controller reconciliation by adding the paused annotation `anywhere.eks.amazonaws.com/paused: true` to the EKS Anywhere cluster, provider datacenterconfig and machineconfig resources, before the components upgrade. After upgrade completes, the annotations are removed so that the cluster controller resumes reconciling the cluster.
+  From `eksctl anywhere` version v0.18.0 onwards, for the management cluster upgrade, CLI relies on eks-a controller to upgrade all the components. If the upgrade process gets stuck and a user fixes underliying issue, the upgrade process will continue. In case if CLI times out during the upgrade process, a user can still fix the underlying issue because the upgrade process is still active since it runs in the controller. 
```

Comment:
```suggestion
  From `eksctl anywhere` version v0.18.0 onwards, for the management cluster upgrade, the CLI relies on the EKS-A controller to upgrade all the components. If the upgrade process gets stuck and a user fixes underlying issue, the upgrade process will continue. This works even if the CLI times out during the upgrade process because the upgrade process is still active since it runs in the controller. 
```